packages/js/src/dashboard/components/problems.js

```diff
@@ -15,6 +15,7 @@ export const Problems = () => {
 	const dismissedProblemsList = useSelect( ( select ) => select( "@yoast/dashboard" ).selectDismissedProblems(), [] );
```

Comment:
Not invented here, but hardcoded string usage while we have a constant and a helper even.

Reply:
Right, I fixed it also other occurences of the same issue.